CRM/Core/ClassLoader.php

```diff
@@ -103,9 +103,11 @@ public function register($prepend = FALSE) {
       return;
     }
     $civicrm_base_path = dirname(dirname(__DIR__));
+    $composer_autoload = "$civicrm_base_path/vendor/autoload.php";
 
-    // @todo: need some way to check if we should register vendor or not
-    //require_once dirname(dirname(__DIR__)) . '/vendor/autoload.php';
+    if (file_exists($composer_autoload)) {
```

Comment:
I just pushed some changes that get `bin/cli.php` bootstrapping Drupal 8 (or at least not crashing when trying to do so). It required creating a `settings_location.php` file because the default logic for finding the Drupal config directory in civicrm.config.php doesn't work with civicrm-core installed by composer. I don't know if there's a sane way to fix that, since I don't know what all the cases in there are for and I don't want to break something